src/django_fast_pagination/pagination.py

```diff
@@ -2,6 +2,7 @@
 Module with custom pagination object
 """
 import sys
+from django.utils.encoding import force_str
```

Comment:
I would leave a blank line between the import sys (standard Python) and the other imports (third-party libs and frameworks)